src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/SvDiscoverFromLocalAssemblyContigAlignmentsSpark.java

```diff
@@ -34,21 +34,50 @@
 import static org.broadinstitute.hellbender.tools.spark.sv.discovery.prototype.AssemblyContigAlignmentSignatureClassifier.RawTypes;
 
 /**
- * This tool takes a SAM file containing alignments of single-ended long read
- * (be it long read sequencing, or contigs assembled from standard Illumina short reads--currently the primary target),
- * searches for reads with--potentially complicated--split alignments indicating the presence of SV breakpoints.
- * In case of complex structural variations (cxSV), also outputs a custom file format containing how affected
- * reference segments are rearranged on the provided sample where the long reads are from.
+ * Parse aligned contigs and call complex structural variants.
+ *
+ * <p>This tool takes a file containing the alignments of assembled contigs or long reads and searches for reads with
+ * potentially complicated split alignments indicating the presence of SV breakpoints.</p>
+ * <p>In the case of complex structural variations (cxSV), it outputs a custom file format containing how affected
```

Comment:
This particular paragraph can go away for now because nothing is actually output yet until #3805 is in, and it has been requested there to output VCF for cxSV as well.